.pylintrc

```diff
@@ -0,0 +1,84 @@
+[MESSAGES CONTROL]
+
+disable=apply-builtin,
```

Comment:
Our maximum line length setting is 120 characters since 80 is too restrictive, can you also disable the 80 character check?

Reply:
That standard is a carry-over from dependency analytics btw.